taskvine/src/examples/vine_example_blast.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2022- The University of Notre Dame
+# This software is distributed under the GNU General Public License.
+# See the file COPYING for details.
+
+# This example shows some of the data handling features of taskvine.
+# It performs a BLAST search of the "Landmark" model organism database.
+# It works by constructing tasks that download the blast executable
+# and landmark database from NCBI, and then performs a short query.
+
+# The query is provided by a string (but presented to the task as a file.)
+# Both the downloads are automatically unpacked, cached, and shared
+# with all the same tasks on the worker.
+
+# Originally written in C (vine_example_blast.c)
+
+from taskvine import *
```

Comment:
Here let's do:
```python
import taskvine as vine
```
and modify the rest of the lines as needed. (In all the examples.)